lib/Monocle.php

```diff
@@ -40,12 +41,27 @@ public function get_active($eids, $swLat, $swLng, $neLat, $neLng, $tstamp = 0, $
             $pkmn_in = substr($pkmn_in, 0, -1);
             $conds[] = "pokemon_id NOT IN ( $pkmn_in )";
         }
-
+        $float = $db->info()['driver'] == 'pgsql' ? "::float" : "";
+        if (!empty($miniv) && !is_nan((float)$miniv) && $miniv != 0) {
+            if (empty($exminiv)) {
+                $conds[] = '((atk_iv + def_iv + sta_iv)' . $float . ' / 45) * 100 >= ' . $miniv;
+            } else {
+                $conds[] = '(((atk_iv + def_iv + sta_iv) / 45)' . $db->info()['driver'] == 'pgsql' ? "::float" : "" . ' * 100 >= ' . $miniv . ' OR pokemon_id IN(' . $exminiv . ') )';
+            }
+        }
+        if (!empty($minlevel) && !is_nan((float)$minlevel) && $minlevel != 0) {
+            if (empty($exminiv)) {
+                $conds[] = '(level >= ' . $minlevel;
```

Comment:
ZeChrales fork doesn't have `level` in DB

Reply:
Removed the level portion of ZeChrales